src/Sylius/Bundle/PayumBundle/Model/PaymentSecurityToken.php

```diff
@@ -81,63 +79,63 @@ public function getDetails()
     /**
      * {@inheritdoc}
      */
-    public function getHash()
+    public function getHash(): string
     {
         return $this->hash;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setHash($hash)
+    public function setHash($hash): void
     {
         $this->hash = $hash;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getTargetUrl()
+    public function getTargetUrl(): string
     {
         return $this->targetUrl;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setTargetUrl($targetUrl)
+    public function setTargetUrl($targetUrl): void
     {
         $this->targetUrl = $targetUrl;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function getAfterUrl()
+    public function getAfterUrl(): string
     {
         return $this->afterUrl;
     }
 
     /**
      * {@inheritdoc}
      */
-    public function setAfterUrl($afterUrl)
+    public function setAfterUrl($afterUrl): void
```

Comment:
It's sadly (this and all above) from `TokenInterface` from `Payum` which does not have any strict types.